spec.emu

```diff
@@ -1842,6 +1851,30 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
             1. Perform ContinueDynamicImport(_payload_, <ins>_moduleRequest_.[[Phase]],</ins> _result_).
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-HostGetModuleSourceRecord" type="host-defined abstract operation">
+        <h1>
+          <ins>
+            HostGetModuleSourceRecord (
```

Comment:
```suggestion
            HostGetModuleRecord (
```

Since source records are not a thing, and this is returning a module record?

Reply:
What do you think about `HostGetSourceModuleRecord` for "get source modulerecord"?

Reply:
Sounds good 👍